src/sentry/sentry_metrics/configuration.py

```diff
@@ -108,6 +111,24 @@ def get_ingest_config(
             )
         )
 
+        _register_ingest_config(
+            MetricsIngestConfiguration(
+                db_backend=IndexerStorage.EXPERIMENRAL,
+                db_backend_options={},
+                input_topic=settings.KAFKA_INGEST_PERFORMANCE_METRICS,
+                output_topic="snuba-generic-metrics-experimental",
```

Comment:
Does this topic exist in SaaS? I am not sure whether you need to create topics in SaaS first or do they get autocreated.

Reply:
This was created in an ops ticket via changes to topictl by @dbanda